src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,43 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Manufacturer manufacturer = new Manufacturer(null, "Tesla Inc.", "USA");
+        manufacturer = manufacturerService.create(manufacturer);
+        Driver firstDriver = new Driver(null, "William Carrington", "345");
+        Driver secondDriver = new Driver(null, "John Bourbon", "346");
+        driverService.create(firstDriver);
```

Comment:
it's a bad practice to pass nulls, create a separate constructor, where you need to pass only next two parameters

Reply:
Interestingly, I did so, but a mentor said it was better to put null values. It was in another problem.